service_test.go

```diff
@@ -266,7 +266,8 @@ func TestCreateServiceWithParentSystem(t *testing.T) {
 
 func TestUpdateService(t *testing.T) {
 	addVars := `{"input":{"description": "The quick brown fox", "framework": "django", "id": "123456789", "lifecycleAlias": "pre-alpha", "name": "Hello World", "parent": {"alias": "some_system"}, "tierAlias": "tier_4"}}`
```

Comment:
🤔 should we revert v1 too since the codeGen messed it up?

Reply:
Yeah I think so. I think it would be confusing to have both with differing struct tags